core/src/main/scala/org/http4s/StaticFile.scala

```diff
@@ -156,7 +156,7 @@ object StaticFile {
 
     def readAll(path: Path, chunkSize: Int): Stream[F, Byte] =
       pulls
-        .fromPath(path, List(StandardOpenOption.READ))
+        .fromPath(path, List(StandardOpenOption.READ))(Stream.syncInstance[F])
```

Comment:
Added this because without it the compiler complained about 2 implicits in scope.
```
[error] /Users/luke.stephenson/projects/http4s/http4s/core/src/main/scala/org/http4s/StaticFile.scala:159:18: ambiguous implicit values:
[error]  both method syncInstance in object Stream of type [F[_]]=> cats.effect.Sync[[β$15$]fs2.Stream[F,β$15$]]
[error]  and value evidence$5 of type cats.effect.Sync[F]
[error]  match expected type cats.effect.Sync[F]
[error]         .fromPath(path, List(StandardOpenOption.READ))
```